sql/src/test/java/ddlog/QueriesTest.java

```diff
@@ -126,7 +126,7 @@ public void testTwoAggregations() {
         String query = "create view v1 as SELECT MIN(column1) + MAX(column1) FROM t1";
         String program = this.header(false) +
                 "typedef Ttmp0 = Ttmp0{col4:signed<64>}\n" +
-                "function agg1(g1: Group<(Tt1)>):Ttmp0 =\n" +
+                "function agg1(g1: Group<'K, (Tt1)>):Ttmp0 =\n" +
```

Comment:
We will probably generate only concrete types here instead of a type variable, but this should work for now.